components/org.wso2.carbon.identity.oauth.uma.resource.service/src/main/java/org/wso2/carbon/identity/oauth/uma/resource/service/ResourceService.java

```diff
@@ -41,4 +41,7 @@ Resource getResourceById(String resourceId)
 
     boolean updateResource(String resourceId, Resource resourceRegistration)
             throws UMAServerException, UMAClientException;
+
+    boolean isResourceBelongsToUser(String resourceId, String userName, String userDomain, String consumerKey)
```

Comment:
would not following method name better?
isResourceOwner(...)

Reply:
Changed the name and updated the PR.